### PR TITLE
Pics

### PR DESCRIPTION
**Pull request by taradactyl27**
added screenshots and gifs to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,32 @@
 
 Transcribe lectures, generate notes, and finally be able to search for exactly which class the professor mentioned that one thing...
 
+# Usage
```
Comment (boubascript) [inline README.md]: Demo seems more accurate

File: README.md [hunk 2]
```diff
@@ -2,6 +2,32 @@
...
+
+Course Page                           |  Lecture Page
+:------------------------------------:|:-------------------------:
+![Course Page](/screenshots/course_page.PNG) | ![Lecture Page](/screenshots/lecture.gif)
+
+Flashcards                            | Recording Page  
+:------------------------------------:|:-------------------------:
+![Flashcards](/screenshots/flashcards.gif) | ![Recording Page](/screenshots/recording.PNG)
+
+Web Screenshot |
```
Comment (boubascript) [inline README.md]: Web "View" instead

**Review by boubascript — APPROVED** [review]
thanks for making these!!
looks fine to me, just have some super nitpicky word choice thoughts you can take em or leave em

File: README.md (hunk 2 above)
Comment (taradactyl27) [reply README.md]: yep, changed to all suggestions. thanks!

**Pull request merged**